osu.Game/Overlays/Profile/Sections/PaginatedProfileSubsection.cs

```diff
@@ -14,18 +14,28 @@
 using osu.Game.Graphics.UserInterface;
 using osu.Game.Online.API;
 using osu.Game.Graphics.Containers;
+using osu.Game.Online.API.Requests;
 using osu.Game.Online.API.Requests.Responses;
 using osuTK;
 
 namespace osu.Game.Overlays.Profile.Sections
 {
     public abstract class PaginatedProfileSubsection<TModel> : ProfileSubsection
     {
+        /// <summary>
+        /// The number of items displayed per page.
+        /// </summary>
+        protected virtual int ItemsPerPage => 50;
```

Comment:
Is there actually a case web-side where this differs?

Reply:
Nope, just made it protected virtual in consistency with the "initial items count" property, can be made constant if you prefer that.

Reply:
Guess it's okay either way, just not sure a scenario this would differ will ever arise.